graal-tools/src/main/scala/io/cloudstate/graaltools/AkkaSerializerRegisterFeature.scala

```diff
@@ -1,66 +0,0 @@
-package io.cloudstate.graaltools
```

Comment:
This feature didn't work as intended, as it could not see what it needed to see, so I removed it until we can find a suitable replacement.

Reply:
The issue here is that not all subtypes are reachable.  But for those that are, it's a convenience to register their constructor (which otherwise mightn't be reached by any bytecode).  (Side note: makes me wonder if registering the classes makes any sense, if the type is already reachable).

And for all the subtypes that are also used but not reached, I would just hard-code them here and reuse the same logic, instead of hard-code them verbosely in the config file.